src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, BoxProps, Skeleton, SxProps } from '@mui/material';
+import { useLocation } from 'react-router-dom';
+import { StyledBaseContributorIndicator } from '../pages/registration/contributors_tab/ContributorIndicator';
+import { dataTestId } from '../utils/dataTestIds';
+import { useProfilePicture } from '../utils/hooks/useProfilePicture';
+import { getContributorInitials } from '../utils/registration-helpers';
+
+interface ProfilePictureProps extends Pick<BoxProps, 'sx'> {
+  personId: string;
+  fullName: string;
+  height: string;
+  hasBorder?: boolean;
+  sx?: SxProps;
+}
+
+export const ProfilePicture = ({ personId, fullName, height, hasBorder, sx }: ProfilePictureProps) => {
+  const { profilePictureQuery, profilePictureString } = useProfilePicture(personId);
+  const isPublicPage = useLocation().pathname.includes('research-profile');
+
+  return (
+    <Box sx={{ height, aspectRatio: '1/1', ...sx }}>
+      {profilePictureQuery.isFetching ? (
+        <Skeleton variant="circular" sx={{ height: '100%' }} />
+      ) : profilePictureQuery.isSuccess ? (
+        <Box
+          component="img"
+          src={profilePictureString}
+          alt="profile-picture"
+          sx={{
+            height: '100%',
+            aspectRatio: '1/1',
+            borderRadius: '50%',
+            border: hasBorder ? '0.125rem solid black' : 'none',
```

Comment:
noe tydelige regler på når/hvor det skal være border eller ikke?

Reply:
Nei, ikke egentlig. Akkurat nå er det kun på `ResearchProfile`, men la det inn som en prop i tilfelle det skla brukes andre plasser. Ser nå at det kanskje er bedre å sende inn en string istedet for å bruke bool :thinking:

Reply:
konklusjon fra designmøte ble å alltid ha border